README.md

```diff
@@ -4,10 +4,84 @@ This repository contains a Javascript implementation of [esptool](https://github
 
 `esptool-js` is based on [Web Serial API](https://wicg.github.io/serial/) and works in Google Chrome and Microsoft Edge, [version 89 or later](https://developer.mozilla.org/en-US/docs/Web/API/Serial#browser_compatibility).
 
+
+## Usage
+
+CDN
+
+``
+
+or import as npm package
+
+`npm install --save esptool-js`
+
+`yarn add --save esptool-js`
+
+Example code:
```

Comment:
I would personally not use TypeScript in the example. However, if you want to, let's mention that it is TS:

```suggestion
Example TypeScript code:
```